appveyor.yml

```diff
@@ -57,26 +71,20 @@ before_build:
     - ps: >-
         gci env:* | sort-object name
 
+# The two 'cmd /c' lines are due to "interesting"
```

Comment:
This comment looks misplaced?

Reply:
Yes, it's a left over from other attempts

Reply:
Removed